hw/ip/rom_ctrl/data/rom_ctrl_testplan.hjson

```diff
@@ -74,8 +76,10 @@
     {
       name: rom_ctrl_check_cg
       desc: '''
-            Collect coverage on the outputs sent to the power manager to confirm
-            that we see pass and fail resutls.
+            - Collect coverage on the outputs sent to the power manager to confirm
+            that we see pass and fail results.
+            - Collect coverage to ensure that a_valid goes high when rom check 
+            is in progerss.
```

Comment:
Nit: prog**re**ss, I think! Maybe also worth saying explicitly that this will ensure TL accesses are blocked until the ROM check is done?